app/utils/parseRecentImage.js

```diff
@@ -0,0 +1,18 @@
+export const parseRecentImage = (data) => {
+  if (!data || data.length === 0) {
+    return null;
+  }
+
+  let result, lastDate;
+
+  data.forEach((item) => {
+    if (!lastDate) {
+      lastDate = new Date(item.earth_date);
+      result = item;
+    }
+
+    if (lastDate < new Date(item.earth_date)) result = item;
+  });
+
+  return result;
```

Comment:
looks like you can sort the list by earth_date and then choose last one?

Reply:
True